container_files/assets/www/doc/builtin/functions/SqlQueryFunction.md

```diff
@@ -6,10 +6,22 @@ The query is parameterized on the function input values.
 SQL query function allows for joins to be implemented
 in MLDB.
 
-Functions created this way executes the given query against the given dataset
-and outputs the `SELECT` expression applied to the first matching
-row (ie, with an `OFFSET` of 0 and a `LIMIT` of 1).  If multiple
-rows are matched, all but the first will be ignored.
+Functions created this way executes the given query against the given dataset.
+
+There are two possibilities for the output:
+
+1.  If the `output` field is set to `FIRST_ROW` (the default), then
```

Comment:
I hear you for examples.

For the enumeration, I will auto-generate the table from the enum documentation.